interpreter/test/start.wast

```diff
@@ -87,6 +89,11 @@
   (start $main)
 )
 
+(module
+  (func $print (import "spectest" "print"))
+  (start $print)
+)
+
```

Comment:
Ah, or did you mean a test where a non-void function is used for start? That exists as well, somewhere in this file.

There is no test that uses an imported function for that, mind you, but the type-checking of the start function should be orthogonal to where the function is coming from. But I'm happy to LGTM it if you want to add another test for that. :)